misc/scripts/package.sh

```diff
@@ -53,6 +53,10 @@ if ((PACSTALL_INSTALL == 0)) && [[ ${pacname} == *-deb ]]; then
     return 0
 fi
 
+if is_package_installed "${gives:-${pacname}}" && dpkg --compare-versions "${pkgver}" eq "$(dpkg-query --showformat='${Version}' --show "${gives:-${pacname}}")"; then
```

Comment:
just realized this doesnt work bc dpkg version is equal to full_version and not pkgver

Reply:
spectacular news though looks like I set it a little down in the file so maybe just run this check after that instead?

https://github.com/pacstall/pacstall/blob/c2fbf25b6693e68d8dbc0eb5a2b0cd16ba44d884/misc/scripts/package.sh#L115-L123

Reply:
Done.